openedx/features/badging/helpers.py

```diff
@@ -116,3 +120,31 @@ def filter_earned_badge_by_joined_team(user, course, earned_badges):
         earned_badge for earned_badge in earned_badges if
         earned_badge.community_id == course_team[TEAM_ROOM_ID_KEY]
     ]
+
+
+def send_user_badge_notification(user, my_badge_url, badge_name):
+    """
+    Send user new badge notification
+    :param user: Notification sender user
+    :param my_badge_url: Redirect url on notification click
+    :param badge_name: Newly earned badge
+    """
+    context = {
+        "badge_name": badge_name
+    }
+
+    body_short = render_to_string('philu_notifications/templates/user_badge_earned.html', context)
+
+    message_type = get_notification_type(
+        NOTIFICATION_TYPE.format(prefix='philu.badging', type='user-badge-earned'))
```

Comment:
Let's either move this to one line or move the closing bracket of `get_notification_type` to next line and properly indent it.

Reply:
done